src/packerlicious/community/builder.py

```diff
@@ -0,0 +1,29 @@
+"""
+Copyright 2018 Matthew Aynalem
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..builder import PackerBuilder
+
+
+class Arm(PackerBuilder):
```

Comment:
any objection to renaming to match the `resource_type`? will make it easier if there are any other arm plugins out there that get added.

```suggestion
class ArmImage(PackerBuilder):
```